dist/fonticonpicker.base-theme.react.css

```diff
@@ -5,15 +5,15 @@
  * React Component to show a picker element to pick font-icons & svg
  * 
  * @author Swashata Ghosh <swashata@wpquark.com>
- * @version 1.2.0
+ * @version 1.2.0-siren-4
  * @link https://github.com/fontIconPicker/react-fonticonpicker
  * @license MIT
  * 
- * Copyright (c) 2021 Swashata Ghosh <swashata@wpquark.com>
+ * Copyright (c) 2022 Swashata Ghosh <swashata@wpquark.com>
```

Comment:
For my own learning. The copyright is updated to reflect that is a new version? And shouldn't left as the most recent year the original author copyrighted it?

Reply:
The year in the distribution files is updated automatically when they are generated as per the original author configuration, the copyright in individual files is preserved in their header.